packages/contracts/test/contracts/discovery/Batch.ts

```diff
@@ -0,0 +1,49 @@
+describe("Batch", () => {
+  describe("constructor", () => {
+    it("works with valid arguments");
+    it("fails with an empty list of projects");
+    it("fails with an address that's not a project");
```

Comment:
How do you propose we check this? Using a role?

Reply:
ERC165